packages/pds/src/api/com/atproto/server/getAccountInviteCodes.ts

```diff
@@ -0,0 +1,103 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { genInvCodes } from './util'
+import { sql } from 'kysely'
+import { InvalidRequestError } from '@atproto/xrpc-server'
+import { nullToZero } from '../../../../db/util'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.server.getAccountInviteCodes({
+    auth: ctx.accessVerifierCheckTakedown,
+    handler: async ({ params, auth }) => {
+      const requester = auth.credentials.did
+      const { includeUsed, createAvailable } = params
+
+      const ref = ctx.db.db.dynamic.ref
+      const [user, userCodesRes] = await Promise.all([
+        ctx.db.db
+          .selectFrom('user_account')
+          .where('did', '=', requester)
+          .select('createdAt')
+          .executeTakeFirstOrThrow(),
+        ctx.db.db
+          .with('use_count', (qb) =>
+            qb
+              .selectFrom('invite_code_use')
+              .groupBy('code')
+              .select(['code', sql<number>`count(*)`.as('uses')]),
```

Comment:
Not necessary at all, but this is another spot the `countAll` helper might fit-in.